frontend/src/components/VSnackbar.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div
-    role="alert"
+    :role="isVisible ? 'alert' : undefined"
```

Comment:
unrelated, but seems fine

Reply:
Agree that it's unrelated, but it's been bugging me every time I test using screenreader: when you search from the homepage, you hear the snackbar read outloud, even if it shouldn't be shown.